samza-shell/src/main/bash/run-class.sh

```diff
@@ -78,15 +83,21 @@ fi
 # permissions for the classpath-related files when they are in their own directory. An example of where
 # this is helpful is when using container images which might have predefined permissions for certain
 # directories.
-CLASSPATH_WORKSPACE_DIR=$base_dir/classpath_workspace
+
+# Note: When on samza-yarn, CLASSPATH_WORKSPACE_DIR looks like:
+# /<hadoop dir>/usercache/<linux account>/appcache/application_1745893616511_0059/container_e64_1745893616511_0059_01_002027/classpath_workspace
+CLASSPATH_WORKSPACE_DIR=$home_dir/classpath_workspace
 mkdir -p $CLASSPATH_WORKSPACE_DIR
+
 # file containing the classpath string; used to avoid passing long classpaths directly to the jar command
 PATHING_MANIFEST_FILE=$CLASSPATH_WORKSPACE_DIR/manifest.txt
+
 # jar file to include on the classpath for running the main class
 PATHING_JAR_FILE=$CLASSPATH_WORKSPACE_DIR/pathing.jar
```

Comment:
Would be great if we can log the pathing_jar_file path and the manifest_file path?

Reply:
Added.